boa_engine/src/bigint.rs

```diff
@@ -389,7 +389,22 @@ impl From<u64> for JsBigInt {
         }
     }
 }
-
+impl From<i128> for JsBigInt {
+    #[inline]
+    fn from(value: i128) -> Self {
+        Self {
+            inner: Rc::new(RawBigInt::from(value)),
+        }
+    }
+}
+impl From<u128> for JsBigInt {
```

Comment:
```suggestion
}

impl From<u128> for JsBigInt {
```

Add new line like other functions.